pallets/teerex/ias-verify/src/lib.rs

```diff
@@ -216,6 +286,114 @@ pub fn verify_ias_report(cert_der: &[u8]) -> Result<SgxReport, &'static str> {
 	parse_report(netscape.attestation_raw)
 }
 
+pub fn parse_ias_report(cert_der: &[u8]) -> Result<SgxEnclaveMetadata, &'static str> {
+	#[cfg(test)]
+	println!("verifyRA: start verifying RA cert");
+
+	let cert = CertDer(cert_der);
+	let netscape = NetscapeComment::try_from(cert)?;
+	let sig_cert = webpki::EndEntityCert::from(&netscape.sig_cert).map_err(|_| "Bad der")?;
+
+	verify_signature(&sig_cert, netscape.attestation_raw, &netscape.sig)?;
+
+	// FIXME: now hardcoded. but certificate renewal would have to be done manually anyway...
+	// chain wasm update or by some sudo call
```

Comment:
add a comment or detail significance here, why `1573419050`?

Reply:
from integritee, we need to figure out what this means.